templates/document.html.slim

```diff
@@ -23,7 +23,9 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
       link rel='stylesheet' href=(attr :revealjs_customtheme) id='theme'
     - else
       link rel='stylesheet' href='#{revealjsdir}/css/theme/#{attr 'revealjs_theme', 'black'}.css' id='theme'
-    include asciidoctor_revealjs.css.slim
+    /! This CSS is generated by the Asciidoctor reveal.js converter to further integrate AsciiDoc's existing semantic with reveal.js
+    style type="text/css"
+      include asciidoctor-default.css
```

Comment:
It is not all just defaults. Maybe we should split into two files then. Upstream defaults and our overrides.

Reply:
The name is misleading but I think we should keep everything in one file to avoid loading multiple files. Performance wise, I think it's better. But we can first add the default styles/rules taken from https://github.com/asciidoctor/asciidoctor/blob/master/data/stylesheets/asciidoctor-default.css and then our overrides?

Reply:
I don't think performance is a strong argument. But I don't think having them separated is that better either. Let's go with what you propose.

Reply:
> I don't think performance is a strong argument.

True :sweat_smile: 

> But I don't think having them separated is that better either. Let's go with what you propose.

Yes less is more and I don't see the value of two files :+1: